src/containers/App/index.jsx

```diff
@@ -1,9 +1,17 @@
 import React, { Component } from "react";
+import AddProducSection from "./../../components/AddProductSection/index";
+
+import AddProduct from "./../AddProductPage/index";
+// import HistoryFromUser HistoryFromUser
+import HistoryFromUser from "./../HistoryFromUserPage/index";
 
 class App extends Component {
   render() {
     return (
       <React.Fragment>
+        {/* <AddProducSection /> */}
```

Comment:
remove comment

Reply:
my app is empty !! how import and comment exist